### PR TITLE
[4.0] administrator/language com_finder

### PR DESCRIPTION
**Pull request by tecpromotion**
Pull Request für Issue #804 .

### Zusammenfassung der Änderungen

rename and 4.0 changes

### CODE REVIEW

**Comment by zero-24** [comment]
WIP Means there is something still open? Or did you already checked for the changes so we just have to rename it?

**Comment by tecpromotion** [comment]
> WIP Means there is something still open? Or did you already checked for the changes so we just have to rename it?

Have a local stash and some strings are not ready. Will change it later for review.

**Comment by tecpromotion** [comment]
@zero-24 your turn 🚀

File: administrator/language/de-DE/com_finder.ini [hunk 1]
```diff
@@ -8,144 +8,105 @@ COM_FINDER="Suchindex"
...
-COM_FINDER_CONFIG_DESCRIPTION_LENGTH_DESC="Die Beschreibung der Suchergebnisse wird nach der eingestellten Anzahl Zeichen abgeschnitten."
-COM_FINDER_CONFIG_DESCRIPTION_LENGTH_DESCRIPTION="Die Beschreibung der Suchergebnisse wird nach der eingestellten Anzahl Zeichen abgeschnitten."
 COM_FINDER_CONFIG_DESCRIPTION_LENGTH_LABEL="Länge der Beschreibung"
-COM_FINDER_CONFIG_ENABLE_LOGGING_DESCRIPTION="Nach Aktivierung dieser Option wird im Protokollverzeichnis (i.&#8239;d.&#8239;R. „logs“) ein Protokoll vom Indexierungsprozess angelegt, um eventuelle Fehler mit diesem besser analysieren zu können.<br /><br /><span style=\"text-decoration: underline; color: lightblue;\"><strong>Hinweis:</strong></span><br />Es wird empfohlen die Protokollierung <strong>nur bei Problemen zu aktivieren</strong>."
-COM_FINDER_CONFIG_ENABLE_LOGGING_LABEL="Protokollierung aktivieren"
-COM_FINDER_CONFIG_EXPAND_ADVANCED_DESC="Legt fest, ob die erweiterten Suchoptionen standardmäßig ausgeklappt werden sollen."
-COM_FINDER_CONFIG_EXPAND_ADVANCED_DESCRIPTION="Legt fest, ob die erweiterten Suchoptionen standardmäßig ausgeklappt werden sollen."
+COM_FINDER_CONFIG_ENABLE_LOGGING_LABEL="Protokollierung aktiviere"
```
Comment (zero-24) [inline administrator/language/de-DE/com_finder.ini]:
```suggestion
COM_FINDER_CONFIG_ENABLE_LOGGING_LABEL="Protokollierung aktivieren"
```

**Pull request merged**